### PR TITLE
update dev config file reader to workaround upstream issue

### PR DESCRIPTION
**Pull request by loadtheaccumulator**
Signed-off-by: Jonathan Holloway <jholloway@redhat.com>

# Description
Workaround for an upstream viper/unmarshal case sensitivity issue when reading a dev config file for local work.
Edits to example podman compose files.
Added a dependency compose file.
Added kafka web UI app for topic and message management in dev
NOTE: working toward consolidating them into a single experience.

Fixes # (issue)

## Type of change

What is it?

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [ ] Refactor

# Checklist:

- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] I run `make pre-commit` to check fmt/vet/lint/test-no-fdo

### CODE REVIEW

**Review by AaronH88 — APPROVED** [review]
/lgtm

File: podman/env/edgemgmt_config.json [hunk 1]
```diff
@@ -3,15 +3,14 @@
...
+                "requestedName": "platform.edge.fleetmgmt.image-build"
             }
         ],
         "topics": [
             {
                 "requestedName": "platform.edge.fleetmgmt.image-build",
-                "name": "platform.edge.fleetmgmt.image-build",
-                "consumerGroupName": "edgemgmt-image-build"
-            }
+                "name": "platform.edge.fleetmgmt.image-build"            }
```
Comment (thom-at-redhat) [inline podman/env/edgemgmt_config.json:13]: should the close brace be on a separate line (to match the rest of the file)?

File: podman/container-compose-dependencies.yml [hunk 2]
```diff
@@ -0,0 +1,45 @@
+version: '2'
+services:
+  postgresql:
+  container_name: postgresql
+    image: registry.redhat.io/rhel8/postgresql-10:1-173.1647451846
```
Comment (thom-at-redhat) [inline podman/container-compose-dependencies.yml:5]: Any reason for Postgres 10?

File: podman/container-compose-dependencies.yml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+    ports:
+      - 5432:5432
+    env_file:
+      - env/edge-api.env
+    volumes:
+      - ~/dev/postgresql:/var/lib/pgsql:Z
+
+  kafka:
+    container_name: kafka
+    image: quay.io/strimzi/kafka:latest-kafka-2.8.1-amd64
```
Comment (thom-at-redhat) [inline podman/container-compose-dependencies.yml:16]: Similarly, do we require this version?  3.2.1 appears to be currnet

File: config/config.go [hunk 4]
```diff
@@ -271,21 +277,19 @@ func Init() {
...
-			fmt.Println("Error reading config file")
-			fmt.Println(configfileErr.Error())
-		}
-		kafkaJSON, err := json.Marshal(options.Get("kafka"))
+		// SOMETHING CHANGED with upstream SetConfigFile or Unmarshal that caused Unmarshal
+		// to freak out on mixedCase being translated to lowercase by SetConfigFile
+		devConfigFile, err := os.ReadFile(configFile)
 		if err != nil {
-			fmt.Println("KafkaConfig marshal error")
+			log.WithField("error", err.Error()).Error("Error reading local dev config file")
```
Comment (thom-at-redhat) [inline config/config.go:285]: Should we include the actual configFile used in the log message?

File: podman/container-compose-dependencies.yml (hunk 2 above)
Comment (loadtheaccumulator) [reply podman/container-compose-dependencies.yml:5]: None other than these are local dev configs being shared that don't need to necessarily stay in sync with the upstream app (yet).

File: podman/container-compose-dependencies.yml (hunk 3 above)
Comment (loadtheaccumulator) [reply podman/container-compose-dependencies.yml:16]: Strictly dev.

**Review by thom-at-redhat — APPROVED** [review]

**Pull request merged**